terraform-modules/stackdriver/k8s-cluster-monitoring/variables.tf

```diff
@@ -0,0 +1,116 @@
+# Required Variables
+
+variable "policy_name" {
+  type        = string
+  description = "Display name to be associated with a particular alerting policy"
+}
+
+variable "condition_combine_method" {
+  type        = string
+  description = "logic to trigger the alert when multiple conditions are present only 'AND' and 'OR' are valid arguments"
+}
+
+variable "project" {
+  type        = string
+  description = "The GCP project the alert policy should be created under"
+}
+
+variable "cluster_name" {
+  type        = string
+  description = " Name of the cluster the alert policy will monitor"
+}
+
+# Optional Variables 
+
+variable "notification_channels" {
+  type        = list(string)
+  default     = []
+  description = "List of channels that should be pinged when alert fires. Array entries must be of form: projects/[PROJECT_ID]/notificationChannels/[CHANNEL_ID]"
+
+}
+
+variable "node_cpu_threshold" {
+  type        = number
+  default     = 0.95
+  description = "The threshold the cpu utilization on a node must be above to trigger the alert. Represents a fraction and must be between 0 and 1"
+}
+
+variable "node_memory_threshold" {
+  type        = number
+  default     = 0.95
+  description = "The threshold memory utilization on a node to trigger an alert. Represents a fraction of allocatble memory and must be between 0 and 1"
+}
+
+variable "node_health_check_threshold" {
+  type        = number
+  default     = 0
+  description = "Will trigger in alert if any of the node health check conditions are in failing or unknown state"
+}
+
+variable "threshold_comparison" {
```

Comment:
This seems like it should be a local variable?

Reply:
👍